edward/criticisms/ppc_plots.py

```diff
@@ -0,0 +1,67 @@
+import seaborn as sns
+
+
+def ppc_density_plot(y, y_rep):
+  """
+  Cretes 1D kernel density plot comparing data to samples from posterior.
```

Comment:
typo; you could also collapse these two lines, e.g.,
```python
"""Creates 1D ...
```
Same goes for docstring below.